pkg/backend/stack.go

```diff
@@ -127,16 +127,53 @@ func GetStackTags() (map[apitype.StackTagName]string, error) {
 			tags[apitype.ProjectDescriptionTag] = *proj.Description
 		}
 
-		if owner, repo, err := gitutil.GetGitHubProjectForOrigin(filepath.Dir(projPath)); err == nil {
-			tags[apitype.GitHubOwnerNameTag] = owner
-			tags[apitype.GitHubRepositoryNameTag] = repo
+		// Add the git metadata to the tags.
+		if err := addGitMetadataToStackTags(tags, projPath); err != nil { // nolint
+			// Intentionally ignore errors adding git metadata to stack tags.
 		}
-
 	}
 
 	return tags, nil
 }
 
+// addGitMetadataToStackTags fetches the git repository from the directory, and attempts to detect
+// and add any relevant git metadata as stack tags.
+func addGitMetadataToStackTags(tags map[apitype.StackTagName]string, projPath string) error {
+	repo, err := gitutil.GetGitRepository(filepath.Dir(projPath))
+	if repo == nil {
+		return fmt.Errorf("no git repository found from %v", projPath)
+	}
+	if err != nil {
+		return err
+	}
+
+	remoteURL, err := gitutil.GetGitRemoteURL(repo, "origin")
+
```

Comment:
Nit: Remove this line of vertical whitespace, and the other one on line 155. So "get `remoteURL`" is one single block.